packages/blazor-workspace/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -162,12 +163,48 @@ export function afterStarted(Blazor) {
     });
 }
 
+function handleRuntimeStarted() {
+    window.NimbleBlazor.hasRuntimeStarted = true;
+}
+
+// Blazor Web Apps
+export function afterWebStarted(Blazor) {
+    registerNimbleEvents(Blazor);
+    // Note: For static SSR, this is the last event called, and hasRuntimeStarted
+    // will remain false.
+}
+
+export function afterServerStarted(_Blazor) {
+    handleRuntimeStarted();
+}
+
+// Blazor Web Apps, WASM Standalone apps with .NET 8
+export function afterWebAssemblyStarted(_Blazor) {
+    if (!window.NimbleBlazor.hasRegisteredEvents) {
+        registerNimbleEvents(Blazor);
+    }
+
+    handleRuntimeStarted();
+}
+
+// Blazor Server/WebAssembly/Hybrid apps
```

Comment:
comment should mention for .net 8 this is only called for blazorhybrid

Reply:
Updated